docs/releases/1.10.txt

```diff
@@ -64,6 +64,8 @@ Minor features
 * :ref:`Distance lookups <distance-lookups>` now accept expressions as the
   distance value parameter.
 
+* Support for SpatiaLite < 3.0 is dropped.
```

Comment:
This goes under "Backwards-incompatible changes" not "Minor features". Sorry I missed it before.